narwhals/_pandas_like/implementations.py

```diff
@@ -0,0 +1,22 @@
+from functools import partial
+from typing import Any
+from typing import Literal
+
+from narwhals.dependencies import Implementation
+from narwhals.dependencies import get_backend
+from narwhals.typing import assert_never
+
+PANDAS_IMPLEMENTATIONS = Literal[
+    Implementation.PANDAS, Implementation.MODIN, Implementation.CUDF
+]
+
+
+def get_series_implementation(implementation: PANDAS_IMPLEMENTATIONS) -> Any:
```

Comment:
not sure about this function name tbh

Reply:
Me neither, feel free to suggest any name